src/components/Chat/MessageWithImage.js

```diff
@@ -185,7 +185,7 @@ const styles = StyleSheet.create({
     marginVertical: 4,
     borderRadius: 8
   },
-  flexlist: {backgroundColor: COLORS.white, borderRadius: 8},
+  flexlist: {backgroundColor: COLORS.almostBlack, borderRadius: 8},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the flexlist has been changed from white to almost black. This change should be reviewed in the context of any text or elements within the flexlist to ensure that there is enough contrast for visibility.
```diff
-  flexlist: {backgroundColor: COLORS.white, borderRadius: 8},
+  flexlist: {backgroundColor: COLORS.almostBlack, borderRadius: 8},
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->